src/Table/TableCell.js

```diff
@@ -72,14 +72,18 @@ function TableCell(props, context) {
     Component = table && table.head ? 'th' : 'td';
   }
 
-  const isTypeHead = variant ? variant === 'head' : table && table.head;
+  let scope = scopeProp;
+  if (!scope && table && table.head) {
```

Comment:
Don't think it is correct. Should work for `variant === 'head'` if set, that's why I had a constant for the same check as for `classes.typeHead` className.

Reply:
That's why you actually had to modify the test as well, but that test was to actually check that `head` table cells have the `scope`.

Reply:
But looking now I'd say it should look like this:

```
let scope = scopeProp;
if (!scope && (variant === 'head' || table && table.head)) {
  scope = 'col';
}
```

Reply:
`variant="head"` is purely UI specific, it won't change DOM structure with the `th` vs `td` usage. For instance, I'm using `td` + `variant="head"` in some cases. So I disagree. It's taking too much assumption.

![capture d ecran 2018-02-14 a 13 52 07](https://user-images.githubusercontent.com/3165635/36205220-4bf8369c-118e-11e8-8a10-bf00475c6d2f.png)

Reply:
Then I don’t have anything else, let’s merge it :)